web/Services/LandlordService.cs

```diff
@@ -30,6 +30,13 @@ public enum ApproveLandlordResult
         ErrorDuplicateMembershipId,
         Success
     }
+    
+    public enum DisableOrEnableLandlordResult

```

Comment:
I would rather if these enums didn't exist on the interface. I think they should either exist on the class implementation of the interface, or in a separate folder :)